### ISSUE TITLE
Wrong Open Api definition in POST /api/v1/contracts/{contractId}/actions

### ISSUE THREAD

**Issue by CarlosLanderas** [Azure-Samples/blockchain#89]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```
Hello, I am generating a C# client with NSwag against the Workbench Api swagger.json and when I am posting an action against a contract the returned type is a raw number representing the action id but the swagger.json specifies that a  **ContractActionOutput** with a **ContractActionID** property is expected so deserialization fails.

Thanks

**Comment by iwooden** [comment]
Thanks for catching this. We will work to fix the generated swagger to make sure it matches the return types of the API. I'll resolve the issue once we release an updated version of Workbench that addresses this.

**Comment by CarlosLanderas** [comment]
@iwooden could you tell me if this issue has been addressed on version 1.6? Thanks!

**Comment by ali92hm** [comment]
Yes it should be in
